phylokit/balance.py

```diff
@@ -26,3 +27,97 @@ def _sackin_index(virtual_root, left_child, right_sib):
 
 def sackin_index(tree):
     return _sackin_index(tree.virtual_root, tree.left_child_array, tree.right_sib_array)
+
+
+@numba.njit(cache=True)
+def _colless_index(postorder, left_child, right_sib):
+    """
```

Comment:
Not much point in adding docstrings to this, I think it's more likely to get out of date and be detrimental than to be of any help.